.github/workflows/python.yml

```diff
@@ -134,69 +201,54 @@ jobs:
                       python/python/tests/pytest_report.html
                       utils/clusters/**
                       benchmarks/results/**
-
-    test-pubsub:
-        runs-on: ${{ matrix.host.RUNNER }}
-        needs: load-engine-matrix
-        timeout-minutes: 35
-        strategy:
-            fail-fast: false
-            matrix:
-                engine: ${{ fromJson(needs.load-engine-matrix.outputs.matrix) }}
-                python:
-                  # - "3.8"
-                  # - "3.9"
-                  # - "3.10"
-                  # - "3.11"
-                    - "3.12"
-                host:
-                    - {
-                        OS: ubuntu,
-                        RUNNER: ubuntu-latest,
-                        TARGET: x86_64-unknown-linux-gnu
-                    }
-                    # - {
-                    #     OS: macos,
-                    #     RUNNER: macos-latest,
-                    #     TARGET: aarch64-apple-darwin
-                    # }
-
-        steps:
-            - uses: actions/checkout@v4
-              with:
-                  submodules: recursive
-        
-            - name: Set up Python
-              uses: actions/setup-python@v4
-              with:
-                  python-version: ${{ matrix.python }}
-        
-            - name: Build Python wrapper
-              uses: ./.github/workflows/build-python-wrapper
-              with:
-                  os: ${{ matrix.host.OS }}
-                  target: ${{ matrix.host.TARGET }}
-                  github-token: ${{ secrets.GITHUB_TOKEN }}
-                  engine-version: ${{ matrix.engine.version }}
-        
-            - name: Test pubsub with pytest
-              working-directory: ./python
-              run: |
-                  source .env/bin/activate
-                  cd python/tests/
-                  pytest --asyncio-mode=auto -k test_pubsub --html=pytest_report.html --self-contained-html
-        
-            - name: Upload test reports
-              if: always()
-              continue-on-error: true
-              uses: actions/upload-artifact@v4
-              with:
-                  name: pubsub-test-report-python-${{ matrix.python }}-${{ matrix.engine.type }}-${{ matrix.engine.version }}-${{ matrix.host.RUNNER }}
-                  path: |
-                      python/python/tests/pytest_report.html
+    test-pubsub-python-client:
```

Comment:
```suggestion
    test-pubsub:
```

Reply:
Will get rid of "client", will leave python